### PR TITLE
Issue 3084 - Ui tweaks for v5

### LINKED ISSUE #3084: CSS tweaks for V5
- [ ] **Add dash array to borders with dashes**
e.g. Adding new Federation or container, empty Federations or containers, empty search results. Dahs should be 5 and gap should be 5.

image

- [x] **Close space between search boxes and buttons on Federation and containers screens**

image
image

- [x] **Remove circle hover state from checkbox on User permissions screen**
No circle needed
image

### PR DESCRIPTION
**Pull request by Amantini1997**
This fixes #3084

#### Description
All George's requests have been address. The checkbox hovering has been address as well, however, the component itself is currently not working (nothing is rendered) and may not be tested

### CODE REVIEW

File: frontend/src/v5/ui/components/shared/dashedContainer/dashedContainer.component.tsx [hunk 1]
```diff
@@ -14,11 +14,20 @@
...
+	dashSize?: number;
+	gapSize?: number;
+};
 
-export const HeaderButtonsGroup = styled.div`
-	& > :last-child {
-		margin-right: 0;
-	}
-`;
+export const DashedContainer = ({ children, ...dashedOptions }: DashedContainerProps) => (
```
Comment (sanmont3drepo) [inline frontend/src/v5/ui/components/shared/dashedContainer/dashedContainer.component.tsx:29]: I would move DashedContainer to controls. Its pretty general.

**Review by sanmont3drepo — CHANGES_REQUESTED** [review]

File: frontend/src/v5/ui/components/shared/dashedContainer/dashedContainer.component.tsx (hunk 1 above)
Comment (Amantini1997) [reply frontend/src/v5/ui/components/shared/dashedContainer/dashedContainer.component.tsx]: Done and I also create the storybook component

**Review by george-hadfield — APPROVED** [review]

**Comment by george-hadfield** [comment]
Approved! Looking smart!

**Pull request merged**